torch_frame/data/tensor_frame.py

```diff
@@ -207,13 +236,37 @@ def __copy__(self) -> 'TensorFrame':
     # Device Transfer #########################################################
 
     def to(self, *args, **kwargs):
-        return self._apply(lambda x: x.to(*args, **kwargs))
+        def fn(x):
+            if isinstance(x, dict):
```

Comment:
```suggestion
            if isinstance(x, Dict):
```

Reply:
`Dict` is for type annotation only, I believe. I don't think that this suggestion will achieve the intended effect

Reply:
Searched around, it seems that types from typing is mainly for hints. Usually in isinstance, concrete types such as `dict` is recommended to be used.